src/modules/upload/index.js

```diff
@@ -36,6 +37,7 @@ const FAILED = 'failed'
 const CONFLICT = 'conflict'
 const QUOTA = 'quota'
 const NETWORK = 'network'
+const FILESIZE = 'Request Entity Too Large'
```

Comment:
```suggestion
const FILESIZE_ERROR = 'Request Entity Too Large'
```

or something like that.

Reply:
✅updated